setup.cfg

```diff
@@ -1,3 +1,68 @@
+[metadata]
+name = safe-eth-py
+version = 4.5.0
```

Comment:
The version changed here. Not sure if this is intended.

Reply:
Yes, it's intended, I wanted to have the version ready asap so I can use it on the tx service and deploy it next week